.github/ISSUE_TEMPLATE/01-new-component.yml

```diff
@@ -0,0 +1,22 @@
+name: New component
+description: Document a component that needs to be added to the GOV.UK Publishing Design Guide
+title: "Document [Enter the name of the component]"
+labels: ["component", "new documentation"]
+projects: ["alphagov/99"]
+body:
+  - type: markdown
+    attributes:
+      value: "# Steps to start documenting a new component"
+  - type: markdown
+    attributes:
+      value: |
+        - Assign yourself to this issue
+        - Under the "Development" select the "Create a new branch" to start documenting the new component
+            - Be sure to check the following criterias:
+                - **Repository destination** is set to `alphagov/govuk-design-guide`
+                - **Branch source** is set to `main`
+            - Select "Checkout locally", which will automatically create a new branch related to this issue
+            - Once selected, copy the command and run it in your VS Code terminal
+            - Confirm that you are working on the new branch by selecting the branch icon in the bottom left corner of VS Code
+        - Start documenting the new component be sure to duplicate the template file `*component-documentation-template` and rename it to the component
```

Comment:
would it be of value to link to the template?

Reply:
Na... From that point the individual should be working in a code editor like VS Code.